pkg/i18n/dutch.go

```diff
@@ -387,7 +402,7 @@ func addDutch(i18nObject *i18n.Bundle) error {
 			Other: `maak een pull-aanvraag`,
 		}, &i18n.Message{
 			ID:    "NoBranchOnRemote",
-			Other: `Deze tak bestaat niet op de afstandsbediening. U moet eerst op de afstandsbediening drukken.`,
+			Other: `Deze tak bestaat niet op de afstandsbediening. U moet het eerst naar de remote pushen.`,
```

Comment:
I think this is a better translations:  
`Deze branch bestaat niet op de remote. U moet het eerst naar de remote pushen.`

Reply:
Consider it done